src/windowedframe.cpp

```diff
@@ -894,7 +894,7 @@ void WindowedFrame::searchText(const QString &text)
             m_focusPos = Search;
         }
         auto temp = text;
-        m_appsManager->searchApp(temp.replace(" ",""));
+        m_appsManager->searchApp(temp.trimmed());
```

Comment:
trimmed : Returns a string that has whitespace removed from the start and the end. 
replace: Replaces every occurrence of the string before with the string after and returns a reference to this string.

Reply:
> for real, I don't remember why I did that, almost a year old or more

I'm sorry for that, maybe we need time to adjust our work to keep up with your pace. We will review the code as much as we can.

Reply:
> trimmed : Returns a string that has whitespace removed from the start and the end. replace: Replaces every occurrence of the string before with the string after and returns a reference to this string.

I think it comes from here
https://github.com/linuxdeepin/dde-launcher/pull/95/commits/463ca29e519b50f0c41ac317e02edf8b98be8eda